web/styles/left_sidebar.css

```diff
@@ -974,6 +974,28 @@ li.top_left_scheduled_messages {
     }
 }
 
+.stream-markers-and-controls {
+    .fa-align-right {
+        color: var(--color-left-sidebar-navigation-icon);
+    }
+
+    &:hover {
+        color: var(--color-text-message-header);
+    }
+}
+
+.bottom_left_row {
+    .fa-align-right {
```

Comment:
Let's add a class `goto-interleaved-stream-view` for this element, rather than referring to it by what icon we picked for it everywhere.